x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/SqlField.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql.plugin;
+
+import org.elasticsearch.common.ParseField;
+
+final class SqlField {
+    public static final ParseField MODE = new ParseField("mode");
```

Comment:
You have those here: https://github.com/elastic/elasticsearch/pull/36149/files#diff-87a0e05a586a10ecca6f36c0e82ef44aR255